docs/man/dune.1.md

```diff
@@ -0,0 +1,291 @@
+**`-s, --start <NODE>`** 
+
+    Start a new node with a given name
+
+**`-c <CONFIG_DIR>, --config <CONFIG_DIR>`**
+
+    Optionally used with --start, a path containing the config.ini file to use.
+
+**`--stop <NODE>`**
+
+    Stop a node with a given name.
+
+**`--remove <NODE>`**
+
+    Remove a node with a given name. If the node is running it will be stopped.
+
+**`--list`**
+
+    Print list of all available nodes and their status.
+
+**`--simple-list`**
+
+    Print the same as --list but without formatting.
+
+**`--set-active <NODE>`**
+
+    Make the given node active.
+
+**`--get-active`**
+
+    Print name of the active node.
+
+**`--export-node <NODE> <PATH>`**
+
+    Export state and blocks log of the given node. PATH may be a directory or a filename with `.tgz` extension.
+    
+    Example: dune --export-node node ~/storage/ 
+
+**`--import-node <NODE> <PATH>`**
+
+    Import state and blocks log to a given node. PATH *must* be a path to a file which contains previously exported 
+    node with `.tgz` extention.
+    
+    Example: dune --import-node node ~/storage/node.tgz`
+
+**`--monitor`**
+
+    Monitor the currently active node.
+
+**`--import-dev-key <KEY>`**
+
+    Import a private key into development wallet.
+
+**`--create-key`**
+
+    Create a public key - private key pair.
+
+**`--export-wallet`**
+
+    Export the development wallet.
+
+**`--import-wallet <DIR>`**
+
+    Import the development wallet.
+
+**`--create-account <NAME> [CREATOR] [PUB_KEY] [PRIV_KEY] [-- OPTIONS]`**
+
+    Create an EOSIO account and an optional creator (the default is eosio).
+    This options calls cleos utility in the Docker container 
+    so you may send additional cleos command line options to the call as OPTIONS
+
+**`--system-newaccount <NAME> [CREATOR] [PUB_KEY] [PRIV_KEY] [-- OPTIONS]`**
+
+    Create an EOSIO account with initial resources using "cleos system newaccount" command.
+    This options calls cleos utility in the Docker container 
+    so you may send additional cleos command line options to the call as OPTIONS
+
+    Example: dune --system-newaccount myaccount -- --buy-ram-bytes 3000
+    
+**`--create-cmake-app <PROJ_NAME> <DIR>`**
+
+    Create a new smart contract project at the given location.
+
+**`--create-bare-app <PROJ_NAME> <DIR>`**
+
+    Create a new empty smart contract project at the given location.
+     
+**`--cmake-build <DIR> [-- OPTIONS]`**
+
+    Build a smart contract project at the given directory.
+    Additional CMake options can be added to CMake call as OPTIONS 
+     
+    Example: dune --cmake-build ~/project -- -DFLAG1=On -DFLAG2=Off               
+
+**`--ctest <DIR> [-- OPTIONS]`**
+
+    Run the ctest tests for a smart contract project at the directory given.
+    Additional ctest options can be added to ctest call as OPTIONS
+     
+    Example: dune --ctest ~/project -- --progress -V
+
+**`--gdb <PROGRAM> [-- OPTIONS]`**
+
+    Start gdb into the container with given executive binary
```

Comment:
```suggestion
    Start gdb in the container with given executive binary
```

Reply:
When I wrote "into" @ScottBailey changed it to "in". He is a native English speaker so I believe he knows better :)